frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -279,8 +279,9 @@ const Map: React.FC<MapProps> = ({
         address: filter.address ?? '',
         pid_pin: filter.pid ?? '',
         property_name: filter.name ?? '',
-        classification:
-          lookupCodes.getClassificationNameById(Number(filter.classificationId)) ?? '',
+        classification: filter.classificationId
+          ? lookupCodes.getClassificationNameById(Number(filter.classificationId))
+          : 'nothing',
```

Comment:
```suggestion
          : '',
```
Empty would probably be better

Reply:
Thanks for catching that, lol......did a cut and paste and forgot to remove that.